stix_shifter_modules/guardium/configuration/lang_en.json

```diff
@@ -39,6 +39,10 @@
             "password": {
                 "label": "Password",
                 "description": "Password of the user with access to the search API"
+            },
+            "logLevel": {
```

Comment:
where doesn't `logLevel ` used in the module? is `logLevel` specific to individual user configuration?